### DISCUSSION TITLE
How to a href?

### DISCUSSION THREAD

**Discussion by wilnoakl** [phcode-dev/phoenix#1831] [Q&A]
Sorry if I am missing something: how do I create links other than manually writing them? E.g. if I want `<a href="https://www.example.com/">Link</a>`can't I select the word Link and then insert the URL and ahref part by right-click or other method?

**Comment by abose** [comment]
By right clicking text in code?

**Comment by wilnoakl** [comment]
![csshtmlvalidator](https://github.com/user-attachments/assets/db7f48bb-2186-4cd6-9cf6-abae278423cd)
This is what I have in mind.

**Comment by abose** [comment]
Got it. Its not supported currently but i think i may have seen some extensions that does that.
